### PR TITLE
docs: improved keywords in index.html file

### PR DESCRIPTION
**Pull request by aryandeelwal**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

**Comment by jeremylt** [comment]
This does not seem to be associated with an open issue. Can you please explain why you believe this change is required?

**Comment by aryandeelwal** [comment]
I thought two lines are repeated so make them in one line by
short-circuiting.

**Comment by jeremylt** [comment]
Again, why is this needed?

Random changes that don't support the projects is not the purpose of Hacktoberfest.

**Review by Sboonny — DISMISSED** [review]
Thank you @aryandeelwal, for trying to refactor the code, as jeremylt said this aren't going to contribute to Hacktoberfest.

If you want to work on this still, can you refactor similar code, in same file.

Here it's https://github.com/freeCodeCamp/freeCodeCamp/blob/9222ca816e4dfaa847b9ec97c099e382863e3580/docs/index.html#L194-L197

and if you can replace the `var` with `let` and `const`, that would be lovely

**Comment by aryandeelwal** [comment]
Thanks for suggesting me as I am new in development i would do my best to
complete this taks

File: docs/index.html
Comment (ieahleen) [inline docs/index.html:212]: I fear that `'Arabic' || 'Hebrew'` will just result in `Arabic`, and we are loosing half of the code

Comment (aryandeelwal) [reply docs/index.html:212]: Review second commit please

Comment (jeremylt) [reply docs/index.html:212]: Both commits are part of the pr - you need to address concerns about the first commit too

Comment (ieahleen) [reply docs/index.html:212]: do not mark requests for change as resolved without resolving them

**Review by majestic-owl448 — DISMISSED** [review]

File: docs/index.html
Comment (aryandeelwal) [reply docs/index.html:212]: How can i resolve it?

Comment (ojeytonwilliams) [reply docs/index.html:212]:
```suggestion
                vm.route.path.search('Arabic') === -1 &&
                vm.route.path.search('Hebrew') === -1
```
To resolve this you can either push a new commit with these changes or click the "Commit suggestion" button

**Review by ojeytonwilliams — DISMISSED** [review]
Thanks @aryandeelwal this LGTM :+1: 

@ieahleen could you merge this if you're happy with the changes?

**Comment by ShaunSHamilton** [comment]
Hello there,

Please edit this PR title to match our recommended title format: https://contribute.freecodecamp.org/#/how-to-open-a-pull-request

**Comment by aryandeelwal** [comment]
docs : improved keywords in index.html file
Can you please confirm first is it an eligible PR title or not?

**Comment by ieahleen** [comment]
@ojeytonwilliams I never looked at this file before 😓 
I'm happy that that line has been ripristinated

Before merging we need to make the linter happy tho

File: docs/index.html
Comment (naomi-lgbt) [inline docs/index.html:175]:
```suggestion
                dynamicText =
```

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]
should fix the linter

**Comment by aryandeelwal** [comment]
Please merge this. All checks have passed.

**Review by naomi-lgbt — APPROVED** [review]

**Review by ojeytonwilliams — APPROVED** [review]

**Comment by ojeytonwilliams** [comment]
Thanks, @aryandeelwal!

**Pull request merged**
